### DISCUSSION TITLE
Font and color for sections with code

### DISCUSSION THREAD

**Discussion by tonge495** [twilio-labs/paste#3158] [Q&A] [answered]
Hi,

I am trying to reskin a knowledge article page to align with the new One Twilio Theme.  I was wondering what font and color to use for the content with code (e.g.` <p class="notification">insert text here</p>`)?

Please see attached for an example of the page.

![Screen Shot 2023-04-12 at 1 50 37 PM](https://user-images.githubusercontent.com/9394413/231813121-43dc2371-e7dd-4d95-bdc0-36aefeb9cd8d.png)

Any help would be appreciated, thank you!

**Comment by SiTaggart** [comment, marked as answer]
Hi @tonge495 

For this I would be reaching for the Inline Code component in the system. https://paste.twilio.design/components/inline-code

Thanks

**Reply by tonge495** [reply]
Awesome thank you!
